LayoutTests/editing/inserting/typing-003-expected.txt

```diff
@@ -745,17 +745,7 @@ EDITING DELEGATE: webViewDidChange:WebViewDidChangeNotification
 EDITING DELEGATE: shouldChangeSelectedDOMRange:range from 232 of #text > DIV > BODY > HTML > #document to 232 of #text > DIV > BODY > HTML > #document toDOMRange:range from 233 of #text > DIV > BODY > HTML > #document to 233 of #text > DIV > BODY > HTML > #document affinity:NSSelectionAffinityDownstream stillSelecting:FALSE
 EDITING DELEGATE: webViewDidChangeSelection:WebViewDidChangeSelectionNotification
 EDITING DELEGATE: webViewDidChange:WebViewDidChangeNotification
-layer at (0,0) size 800x600
-  RenderView at (0,0) size 800x600
-layer at (0,0) size 800x600
-  RenderBlock {HTML} at (0,0) size 800x600
-    RenderBody {BODY} at (8,8) size 784x584
-      RenderBlock {DIV} at (0,0) size 784x140 [border: (2px solid #FF0000)]
-        RenderText {#text} at (14,14) size 738x112
-          text run at (14,14) width 702: "xxxx x xx xxxx xxxxxx xxxxx xxxxxx xxx xx xxxx xxx xxx xx x xxxxx "
-          text run at (14,42) width 738: "xxxxxxxxxxxx xxxxxxxx xxx xxxx xxxxx xx xxx xxxx xxxx xxxxx xxxxxx "
-          text run at (14,70) width 720: "xxx xx xxxx xxx xxx xx xxxxxx xxxxx xxxxxx xxx xx xxxx xxx xxx xx x "
-          text run at (14,98) width 354: "xxxxx xxxxxxxxxxxx xxxxxxxx xxx"
-          text run at (368,98) width 6: " "
```

Comment:
So it looks like one aspect of this test that might be lost in the transition to `dump-as-markup.js` are the words on each individual text run, which seem (*almost*) consistent across the different baselines — only in the GTK result, the offsets of text runs 2-5 seem to be different. Do you think it's worth preserving this part of the test? (potentially, in such a way that it would only check what text is present in each of the 5 text runs).

Reply:
I don't think we need to preserve that. As far as I can tell, the core of this test is the typed text matching the expectation, not so much that specific line contains specific characters.